tools/topology/topology2/cavs-es83x6.conf

```diff
@@ -0,0 +1,249 @@
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:platform>
+<searchdir:platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<virtual.conf>
+<host-gateway-playback.conf>
+<host-gateway-capture.conf>
+<io-gateway.conf>
+<io-gateway-capture.conf>
+<host-copier-gain-mixin-playback.conf>
+<mixout-gain-dai-copier-playback.conf>
+<deepbuffer-playback.conf>
+<dai-copier-be.conf>
+<dai-copier-eqiir-module-copier-capture.conf>
+<gain-capture.conf>
+<gain-module-copier.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<dmic.conf>
+<hda.conf>
+<intel/hw_config_cardinal_clk.conf>
+<manifest.conf>
+<route.conf>
+<intel/common_definitions.conf>
+<dai-copier.conf>
+<module-copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+<dmic-default.conf>
+<hdmi-default.conf>
+<hdmi-in-default.conf>
+IncludeByKey.HDMI_IN_CAPTURE {
+	"true"	"platform/intel/hdmi-in-capture.conf"
+}
+<input_pin_binding.conf>
+<output_pin_binding.conf>
+<input_audio_format.conf>
+<output_audio_format.conf>
+
+Define {
+	MCLK 				24576000
+	# override DMIC default definitions
+	NUM_DMICS			0
+	DMIC0_ID			1
+	DMIC1_ID			2
+	DMIC0_HOST_PIPELINE_ID		13
+	DMIC0_DAI_PIPELINE_ID		14
+	DMIC0_HOST_PIPELINE_SINK	'gain.13.1'
+	DMIC0_DAI_PIPELINE_SRC		'module-copier.14.2'
+	DMIC0_DAI_COPIER		'dai-copier.DMIC.dmic01.capture'
+	DMIC0_DAI_GAIN			'eqiir.14.1'
+	DMIC0_NAME			'dmic01'
+	DMIC0_PCM_CAPS			'Gain Capture 13'
+	# override BE link ids
+	NUM_HDMIS			3
+	HDMI1_ID			3
+	HDMI2_ID			4
+	HDMI3_ID			5
+	PLATFORM 			"none"
+	USE_CHAIN_DMA			"false"
+	HEADSET_CODEC_NAME		'SSP1-Codec'
+	HEADSET_PLAYBACK_PCM_STREAM_NAME 'Headset Playback'
+	HEADSET_CAPTURE_PCM_STREAM_NAME	'Headset Capture'
+	HEADSET_SSP_DAI_INDEX		1
+	HEADSET_HW_CONFIG_NAME		'HEADSET HWCFG'
+	HEADSET_PCM_NAME		"Headset"
+	HEADSET_PCM_ID			0
+}
+
+# override defaults with platform-specific config
+IncludeByKey.PLATFORM {
+	"mtl"	"platform/intel/mtl.conf"
+}
+
+# include DMIC config if needed.
+IncludeByKey.NUM_DMICS {
+	"[1-4]"	"platform/intel/dmic-generic.conf"
+}
```

Comment:
And here as well if NUM_DMICS is always 0. If you want to keep it for later configurations with dmic, that is different then.

Reply:
done. updated the latest patch. please check.